keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageServiceTests.kt

```diff
@@ -196,13 +199,6 @@ class ImageServiceTests : JUnit5Minutests {
         expectThat(newestImage.imageName).isEqualTo("my-package-0.0.1_rc.99-h100")
       }
 
```

Comment:
is this test no longer relevant?

Reply:
I think I removed this by mistake. Will undo.